include/eve/module/core/regular/impl/simd/x86/scan.hpp

```diff
@@ -79,18 +79,20 @@ use_scan_in_lanes(Wide)
   else return std::false_type {};
 }
 
-template<simd_value Wide, typename Op, typename Zero>
-EVE_FORCEINLINE auto
-scan_(EVE_SUPPORTS(avx2_), Wide v, Op op, Zero z_) requires(current_api == avx2)
+template<callable_options O, typename T, typename N, typename Op, typename Zero>
```

Comment:
There are no requirements for op and zero. I thought in new format we require full requirements.

Reply:
the requirement is not here, it's on the operator()